admin/test/features/step_definitions/generate_pupil_pins_steps.rb

```diff
@@ -195,6 +195,13 @@
   expect(generate_pupil_pins_page.find_pupil_row(@pupil_name).pin.text).to eql pupil_pin.to_s
 end
 
+Then(/^check form should be assigned to the pupil$/) do
+  pupil_upn = @stored_pupil_details['upn'].to_s
+  pupil_id = SqlDbHelper.pupil_details(pupil_upn)['id']
+  check_entry = SqlDbHelper.check_details(pupil_id)
+  expect(check_entry['checkForm_id'].nil?).to be_falsey, "Check Form is not assigned to the Pupil when pin is generated"
+end
```

Comment:
@sinhaamit1980 To make this test potentially even better, is it worth maybe going to the 'checkForm' table with the check_entry['checkForm_id'] and the check assigned to the pupil is not deleted?